src/components/pages/Partners.jsx

```diff
@@ -59,10 +59,28 @@ class Partners extends React.Component {
             </div>
           </section>
 
-          <section className="l-section">
+          <section className="l-section partners -first">
             <div className="l-container">
               <div className="row">
-                {this.props.list.map((p, i) => <PartnerBlock key={i} item={p} />)}
+                <div className="column small-12">
+                  <h2 className="title c-text -huger -primary -thin">Founding partners</h2>
+                </div>
+              </div>
+              <div className="row">
+                {this.props.featured.map((p, i) => <PartnerBlock key={i} item={p} />)}
+              </div>
+            </div>
+          </section>
+
+          <section className="l-section partners">
```

Comment:
`-partners` would be better as long as it will modify the `l-section`. What do you think?